packages/components/src/base_tooltip/constants.jsx

```diff
@@ -10,4 +10,4 @@ export const TRIGGER_HOVER = 'hover';
 export const TRIGGER_CLICK = 'click';
 export const TRIGGER_DELAYED = 'delayed';
```

Comment:
if you globally export it -> rename it 
if you don't globally export it -> local naming 

Since you export the whole file -> rename it